dask_geomodeling/tests/test_geometry.py

```diff
@@ -1013,6 +1014,28 @@ def test_overlapping_geometries(self):
         result = view.get_data(**self.request)
         self.assertEqual(result["features"]["agg"].values.tolist(), [36.0, 18.0])
 
+    def test_aggregate_percentile_one_empty(self):
+        # after bug report BACK-720: NaN values get replaced with some other
+        # value in case of percentile statistics
```

Comment:
I think we should not refer to our internal issue tracking system in this open package here.